lib/sinon/behavior.js

```diff
@@ -180,6 +181,15 @@ var proto = {
         return this.returnValue;
     },
 
+    effectiveWrappedMethod: function effectiveWrappedMethod() {
+        for (var stubb = this.stub; stubb; stubb = stubb.parent) {
+            if (stubb.wrappedMethod) {
+                return stubb.wrappedMethod;
+            }
+        }
+        return undefined;
```

Comment:
I'd much prefer failing with an explit `Error` here telling us which assumption was wrong, rather than failing in the next step when ` wrappedMethod.apply` results in a null pointer exception.

```suggestion
        throw new Error("Unable to find wrapped method");
```

Reply:
I had thought about that, but hadn't done it at first in the interest of minimizing changes to existing behavior.  But if you're in favor of it, then so am I :)